Jellyfin.Server/Jellyfin.Server.csproj

```diff
@@ -12,7 +12,8 @@
     <!-- We need C# 7.1 for async main-->
     <LangVersion>latest</LangVersion>
     <!-- Disable documentation warnings (for now) -->
-    <NoWarn>SA1600;CS1591</NoWarn>
+    <NoWarn>SA1600;SA1601;CS1591</NoWarn>
```

Comment:
is `SA1601` also a documentation warning?

Reply:
Yep